goss/goss-windows.yaml

```diff
@@ -1,14 +1,107 @@
 command:
+  Powershell:
+    exec: C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe -command "(Get-Host).Version"
+    exit-status: 0
+  az:
+    exec: C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\wbin\az.cmd version
+    exit-status: 0
+    stdout:
+      - 2.54.0
+  chocolatey:
+    exec: C:\ProgramData\Chocolatey\bin\choco.exe
+    exit-status: 0
+    stdout:
+      - 1.4.0
+  chocolatey-awscli:
```

Comment:
Is there a reason to prefix with `chocolatey`?

Reply:
except to point out the fact that it's installed by chocolatey none ... and will be best without planning the factorization ...